libs/libcommon/src/libcommon/constants.py

```diff
@@ -24,12 +24,11 @@
 PROCESSING_STEP_DATASET_OPT_IN_OUT_URLS_COUNT_VERSION = 2
 PROCESSING_STEP_DATASET_PARQUET_VERSION = 2
 PROCESSING_STEP_DATASET_SIZE_VERSION = 2
-PROCESSING_STEP_PARQUET_AND_DATASET_INFO_VERSION = 2
 PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_PARQUET_VERSION = 2
 PROCESSING_STEP_CONFIG_PARQUET_AND_INFO_VERSION = 3
 PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_STREAMING_VERSION = 3
 PROCESSING_STEP_SPLIT_NAMES_FROM_DATASET_INFO_VERSION = 3
-PROCESSING_STEP_SPLIT_NAMES_FROM_STREAMING_VERSION = 3
+PROCESSING_STEP_CONFIG_SPLIT_NAMES_FROM_STREAMING_VERSION = 4
```

Comment:
why do we upgrade the version? If I understand correctly, the job runner in itself has not changed, right? ie. what would we gain by running it again on all the splits?

Reply:
ah true, thanks for noticing! changed it